libraries/apollo-tooling/src/main/kotlin/com/apollographql/apollo3/tooling/SchemaUploader.kt

```diff
@@ -11,45 +14,87 @@ object SchemaUploader {
       key: String,
       graph: String?,
       variant: String = "current",
+      subgraph: String? = null,
+      revision: String? = null,
       headers: Map<String, String> = emptyMap(),
```

Comment:
FWIW, this is used in Confetti [here](https://github.com/joreilly/Confetti/blob/main/backend/service-graphql/src/main/kotlin/dev/johnoreilly/confetti/backend/DefaultApplication.kt#L124). Will be interesting to see if still working.